### DISCUSSION TITLE
UX: Unnecessary  Misleading 401 Error and/or Re-login after updates

### DISCUSSION THREAD

**Discussion by TiancongLx** [open-webui/open-webui#7486] [General]
## Description

After each update, users are experiencing two inconvenient behaviors:

1. Forced re-authentication for all users
2. Display of a misleading __RED__ 401 error message

## Current Behavior

- Users are automatically logged out after updates
- A __RED__ 401 error message is displayed, suggesting an authentication error
- This creates unnecessary anxiety for users, as no actual error has occurred

## Expected Behavior

- Preserve user sessions updates when possible
- If re-authentication is absolutely necessary, show a friendly message explaining why
- Remove the __RED__ 401 error message and replace it with a more appropriate update notification

## Impact

- Degraded user experience
- Unnecessary user anxiety due to error messages
- Increased support inquiries about "errors" that aren't actual errors

One of my friends using my hosted OI daily, got confused for the red error.
> "Did I do something wrong? Why?"
> "There, there. It's just a normal new version release, and after a re-login it'll be OK. The `401 error code` is meaning ..."

## Technical Details

- Affects all users regardless of their authentication status
- Current implementation treats version changes as authentication failures

## Proposed Solutions

- Replace the error message with a user-friendly update notification
- Add graceful session restoration logic

**Comment by tjbck** [comment]
https://docs.openwebui.com/getting-started/advanced-topics/env-configuration#webui_secret_key
